modules/dop/initialize.go

```diff
@@ -183,6 +184,9 @@ func (p *provider) Initialize() error {
 		}
 	}()
 
+	// update pipeline cms
+	go updateCmsNsConfigs(ep)
```

Comment:
use a better name to clarify your purpose.

Reply:
error was ate.

Reply:
done